src/test/java/core/SlaveTest.java

```diff
@@ -46,9 +45,7 @@ public void setUp() throws ExecutionException, InterruptedException {
         slave = slaveController.install(jenkins).get();
     }
 
-    /**
-     * Scenario: Bring slave offline and then online.
-     */
+    // Bring slave offline and then online.
```

Comment:
use `/** */` so that it gets javadoc formatted and then the IDE can give you the doc,